_rules/user-motion-disabled-ed12b1.md

```diff
@@ -0,0 +1,574 @@
+---
+id: ed12b1
+name: User motion can be disabled
+rule_type: atomic
+description: |
+  This rule checks that functionality that can be operated by user motion, responding to the motion can be disabled to prevent accidental actuation.
+accessibility_requirements:
+  wcag21:2.5.4: # Motion Actuation (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+  - Accessibility tree
+acknowledgements:
+  authors:
+    - João Vicente
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML document][] that has [functionality][] that can be operated by user motion.
+
+## Expectation
+
+For the test target, user motion [can be disabled][] and the control for disabling it must be a [user interface component][] and it should be [visible][] and [included in the accessibility tree][] with an [accessible name][] that is not empty ("").
+
+## Assumptions
+
+- This test assumes that the motion to operate [functionality][] is not used through an [accessibility supported][] interface
+- This test assumes that motion is not [essential][] for the [functionality][]
+
+## Accessibility Support
+
+- Device [orientation events][] may include sensitive data. Implementations must fire events only on [secure browsing contexts][]
+
+## Background
+
+- [Understanding Success Criterion 2.5.4: Motion Actuation](https://www.w3.org/WAI/WCAG21/Understanding/motion-actuation.html)
+- [G213: Provide conventional controls and an application setting for motion activated input](https://www.w3.org/WAI/WCAG21/Techniques/general/G213.html)
+- [DeviceOrientation Event Specification](https://www.w3.org/TR/orientation-event/)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+The [HTML document][] has [functionality][] that can be operated by user motion and [can be disabled][], and the control for disabling is [visible][] and is [included in the accessibility tree][] with an [accessible name][] that is not empty ("").
+
+```html
+<html>
+  <head>
+    <title>Passed Example 1</title>
+    <style>
+      div:first-child {
+        font-size: 1.5em;
+        text-align: center;
+        vertical-align: middle;
+        display: table-cell;
+        height: 50vh;
+        width: 100vw;
+      }
+      #target {
+        background: white;
+        border: 1px solid black;
+      }
+    </style>
+    <script>
```

Comment:
Code seems to be shared by all Passed/Failed Example, except Failed Example 1 (canot be disable). Again, this can be refactored in a test asset and `init` can take an `options` parameter to handle the `canBeDisabled` part. Maybe rename it `activateColorPinch` or something similar to make the intend more obvious.